application/helpers/expressions/em_cache_helper.php

```diff
@@ -144,13 +144,23 @@ protected static function getSurveyCache()
     }
 
     /**
-     * True if all conditions are set to use the emcache.
+     * True if all conditions are met to use the emcache.
      *
      * @return boolean
      */
     public static function useCache()
     {
-        // If forced, always use.
+        // Never use from CLI.
+        if (php_sapi_name() === 'cli') {
```

Comment:
Since we rae in Yii, maybe `Yii::app() instanceof CConsoleApplication` ?

Reply:
Not sure that works with PHPUnit.

Reply:
Wait, I needed to remove it to make it testable. See updated code.